src/main/java/io/openliberty/tools/common/plugins/util/DevUtil.java

```diff
@@ -1801,6 +1818,15 @@ public void runHotkeyReaderThread(ThreadPoolExecutor executor) {
                                 info(formatAttentionMessage("To restart the server, type 'r' and press Enter."));
                             }
                             info(formatAttentionMessage("To stop the server and quit dev mode, press Ctrl-C or type 'q' and press Enter."));
+                            if (httpPort != null) {
+                                info(formatAttentionMessage("Liberty local http port: " + httpPort));
+                            }
+                            if (httpsPort != null) {
+                                info(formatAttentionMessage("Liberty local https port: " + httpsPort));
+                            }
```

Comment:
We should use a different message depending on container mode or not.

Consider the term "host port" or "Docker host port" to be consistent with [Docker documentation](https://docs.docker.com/network/links/).